corehq/apps/app_manager/app_schemas/form_metadata.py

```diff
@@ -221,15 +221,26 @@ def _save_to_case_root_node(self, form_unique_id, question):
         self._seen_save_to_case[form_unique_id].append(question_path)
         return response
 
-    def _serialized_question(self, form_unique_id, question):
+    def _serialized_question(self, form_unique_id, question, actions):
```

Comment:
Does this get called for both regular and advanced forms?

Reply:
Realized I'm not certain the conditional case update itself applies to both regular and advanced forms, but I think it does.

Reply:
I'm not really familiar with advanced forms - a quick test locally says yes it gets called, but no this does not work with them yet. The list of actions looks like it's structured differently (`load_update_cases` vs `update_case`).